test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -2951,6 +2965,39 @@ func createAndWaitReadyNamespacedWithWarnHandler(t *testing.T, client clientset.
 	return nil
 }
 
+func cleanupPolicy(t *testing.T, client clientset.Interface, policy *admissionregistrationv1.ValidatingAdmissionPolicy, binding *admissionregistrationv1.ValidatingAdmissionPolicyBinding) error {
```

Comment:
I see, so cleanupPolicy ensures there are not objects on apiserver, but does not guarantee the admission plugin has seen those changes, right?

Reply:
Yeah that's right, the [policy_source's cache](https://github.com/kubernetes/kubernetes/blob/release-1.30/staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/generic/policy_source.go#L253-L363) needs to be updated after a new policy is added in the test in order for the admission plugin to actually see those policies.